CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Airbrake Changelog
 
 ### master
 
+* Fixed Logger integration error and refactored the way it works. It's no longer
```

Comment:
`Fixed Logger integration error and refactored the way it works` provides 0 information.